packages/solid/tsconfig.json

```diff
@@ -1,8 +1,13 @@
 {
   "extends": "./node_modules/@vercel/ai-tsconfig/react-library.json",
   "compilerOptions": {
-    "target": "ES2018",
-    "stripInternal": true
+    "jsx": "preserve",
+    "jsxImportSource": "solid-js",
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
+    "target": "ESNext",
+    "module": "ESNext",
+    "stripInternal": true,
+    "moduleResolution": "Bundler"
```

Comment:
@ian-pascoe why were these changes needed?

Reply:
Package resolution for SSR types was off, these are the recommended settings for SolidJS